client/src/app/deck/deck.module.scss

```diff
@@ -167,6 +167,46 @@
 		height: 1.5rem;
 		filter: drop-shadow(2px 2px 0 rgba(0, 0, 0, 25%));
 	}
+
+	&.farm {
+		background-color: rgb(124, 204, 12);
+	}
+
+	&.redstone {
+		background-color: rgb(185, 33, 42);
+	}
+
```

Comment:
don't copy all of this CSS twice. Its definitely going to be forgot about and become desynced.

Reply:
Oops, was going to do this, fixed